manifests/m/MITMediaLab/Scratch/3/3.24.0/MITMediaLab.Scratch.3.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MITMediaLab.Scratch.3
+PackageVersion: 3.24.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://downloads.scratch.mit.edu/desktop/Scratch%203.24.0%20Setup.exe
+  InstallerSha256: 985F33F2B87C059CF05C3517DFCB0905B1A8385FC327C82FB1282AD22486B191
+  UpgradeBehavior: install
```

Comment:
```suggestion
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://downloads.scratch.mit.edu/desktop/Scratch%203.24.0%20Setup.exe
  InstallerSha256: 985F33F2B87C059CF05C3517DFCB0905B1A8385FC327C82FB1282AD22486B191
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```

Reply:
Are you sure that works? I tried it and it didn't...

Reply:
Requires Administrator
![image](https://user-images.githubusercontent.com/15158490/126673843-001df261-859f-4a7a-8ec2-6908c0d4f500.png)

Does not require Administrator
![image](https://user-images.githubusercontent.com/15158490/126673898-6ec5d564-69bc-41b2-907e-b5492de2b88c.png)

After testing:
The issue with the Machine one is that it requires Powershell to be run as Administrator
`.\Scratch3.24.0Setup.exe /S /ALLUSERS` without admin = nothing
`.\Scratch3.24.0Setup.exe /S /ALLUSERS` as admin = installs